crates/tessera-backbone/Cargo.toml

```diff
@@ -0,0 +1,19 @@
+[package]
+name = "tessera-backbone"
+version.workspace = true
+edition.workspace = true
+authors.workspace = true
+repository.workspace = true
+keywords.workspace = true
+categories.workspace = true
+
+[dependencies]
+tokio = { workspace = true, features = [ "full" ] }
+anyhow = { workspace = true }
+tracing = { workspace = true }
+tracing-subscriber = { version = "0.3.18", features = ["json"] }
```

Comment:
Since tracing-subscriber is likely to be commonly used, would it make sense to reference it as a workspace dependency?

Reply:
It makes sense. i will move this dependency into workspace dependencies.